catalog/content/core/content-core-catalogerplugin/pom.xml

```diff
@@ -81,14 +90,13 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>guava</Embed-Dependency>
+                        <Embed-Dependency>ddf-security-common,platform-util-unavailableurls</Embed-Dependency>
                         <Private-Package>
                             ddf.content.resource.impl,
                             ddf.catalog.data.impl,
```

Comment:
Not really related, but might be possible to embed this deps instead of private packaging. may be missing something of why they're Private packaged though